cli/lib/util/validate_input.dart

```diff
@@ -0,0 +1,34 @@
+import 'package:cli/exceptions/exceptions.dart';
+
+class ValidateUtil {
+  static void validateName(String? itemName) {
+    if (itemName == null) {
+      throw ItemNullException();
+    }
+    if (itemName.length < 2) {
+      throw ItemNameException();
+    }
+  }
+
+  static void validateType(String? itemType) {
+    if (itemType == null) {
+      throw ItemNullException();
+    }
+    bool isValid = (itemType == 'raw') ||
+        (itemType == 'manufactured') ||
+        (itemType == 'imported');
```

Comment:
Defile all the types as constants somewhere and use those constants everywhere needed